jwst/mrs_imatch/mrs_imatch_step.py

```diff
@@ -398,9 +408,20 @@ def _match_models(models, channel, degree, center=None, center_cs='image'):
 
 def _find_channel_bkg_index(model2d, channel):
     """
-    Return the index of the background subschema corresponding to a given
-    channel.
+    Find index of the background subschema for a given channel.
+
+    Parameters
+    ----------
+    model2d : DataModel
+        2D data model.
```

Comment:
I know the parameter is called `model2d` but it looks like this function would operate on any model that has `meta.background.polynomial_info`